### PR TITLE
More fine tuning of JavaScript doc

### PR DESCRIPTION
**Pull request by waderyan**

### CODE REVIEW

File: docs/languages/javascript.md [hunk 1]
```diff
@@ -108,17 +110,17 @@ npm install typings --global
...
-typings search lodash
+# Replace "node" with the name of your Node.js module
+typings search node
 ```
 
 Install the typings at the same location as `jsconfig.json` (the root of your JavaScript project).
 
 ```bash
-typings install lodash --save
+typings install dt~node --global
```
Comment (gregvanl) [inline docs/languages/javascript.md]: Shouldn't we continue to recommend the --save switch?

File: docs/languages/javascript.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ There are two more steps to configure IntelliSense across all the files in your
...
+For the details of how JavaScript IntelliSense works, including being based on type inference, JsDoc annotations, TypeScript declarations, and mixing JavaScript and TypeScript projects, see the [JavaScript language service documentation](https://github.com/Microsoft/TypeScript/wiki/Salsa). When type inference does not provide the desired information, type information may be provided explicitly with JSDoc annotations. This document describes the [JSDoc annotations](https://github.com/Microsoft/TypeScript/wiki/JsDoc-support-in-JavaScript) currently supported. In addition to objects, methods, and properties, JavaScript IntelliSense window also provides basic word completion based on the other words in the file.
```
Comment (gregvanl) [inline docs/languages/javascript.md]: ", the JavaScript IntelliSense window also provides" or ", JavaScript IntelliSense also provides"

File: docs/languages/javascript.md [hunk 3]
```diff
@@ -108,17 +110,17 @@ npm install typings --global
...
 
 Install the typings at the same location as `jsconfig.json` (the root of your JavaScript project).
 
 ```bash
-typings install lodash --save
+typings install dt~node --global
 ```
 
-You can learn more in the documentation for [typings](https://www.npmjs.com/package/typings) and at [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped), which is a repository of typings files for many major JavaScript libraries and environments.
+The flag `--global` is for libraries on the global (`window.*`) object. The prefix `dt~` specifies the source of the Node.js typings file. You can see the sources, versions, and updated date information using `typings search node`. Learn more in the documentation about typings in the [documentation](https://www.npmjs.com/package/typings) and at [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped), which is a repository of typings files for many major JavaScript libraries and environments.
```
Comment (gregvanl) [inline docs/languages/javascript.md]: Learn more about typings in the [Typings documentation] and at..

File: docs/languages/javascript.md (hunk 1 above)
Comment (waderyan) [inline docs/languages/javascript.md]: Done.

File: docs/languages/javascript.md (hunk 2 above)
Comment (waderyan) [inline docs/languages/javascript.md]: Done.

File: docs/languages/javascript.md (hunk 3 above)
Comment (waderyan) [inline docs/languages/javascript.md]: Done.

**Pull request merged**